appveyor.yml

```diff
@@ -128,11 +118,11 @@ for:
       - "pylint --rcfile .pylintrc samcli"
       # There are some functional tests that are currently broken due to not being updated with changed code or still running with node4.3 runtimes
       # We need to update those but this allows us to at least runs the ones we currently have working
-      - "pytest tests/functional/commands/validate tests/functional/commands/cli/test_global_config.py"
+      - "pytest -n 4 tests/functional/commands/validate tests/functional/commands/cli/test_global_config.py"
 
       # Runs only in Linux
       - sh: "pytest -vv tests/integration"
```

Comment:
in the future, we should parallelize this too

Reply:
Agreed. I tried to do that in this PR but required some additional changes in the build tests. We need to fix this part of the tests: https://github.com/awslabs/aws-sam-cli/blob/develop/tests/integration/buildcmd/test_build_cmd.py#L72 because we validate the image was deleted. I think we did this to ensure new tests would all start with downloading images and from the same place. While I agree, this actually added mins to the test runs as well. I was going to try and run all other integ tests but build but wanted to try and keep things scoped and not solve everything at once here.

Reply:
Correction: That method verifies we are deleting the container not the image.